README.md

```diff
@@ -14,98 +14,220 @@ Instagram - [@chaynhq](https://www.instagram.com/chaynhq/)
+
+**The first time you run the project, create the Bloom database**
+
+```
+CREATE DATABASE bloom WITH OWNER = postgres ENCODING = 'UTF8' CONNECTION LIMIT = -1;
+```
+
+```
+cat ./bloom_db_local.sql | docker exec -i bloom-local-db pg_restore --verbose --clean --no-acl --no-owner -U postgres -d bloom
+```
 
-Swagger automatically reflects all the endpoints that have been created. It also gives details of an api's url and exposes a JSON object needed to make a request and also what an api's response looks like.
+**Run the app**
 
-To access Swagger simply run the project and visit:
+```bash
+yarn start:dev
+```
+
+You should see this in the shell output:
 
 ```shell
-http://localhost:35001/api/
+Listening on localhost:35001, CTRL+C to stop
+```
+
+### Database migrations
+
+A migration in TypeORM is a single file with SQL queries to update a database schema as updates/additions are made. Read more about migrations [here](https://github.com/typeorm/typeorm/blob/master/docs/migrations.md).
+
+Migrations are automatically run when the app is built docker (locally) or Heroku for staging and production apps.
+
+**You'll need to generate and run a migration each time you add or update a database field or table.**
+
+To generate a new migration
+
+```bash
+yarn migration:generate -- bloom_backend
+```
+
+To run (apply) migrations
+
+```bash
+yarn migration:run
 ```
 
-## Firebase
+To revert a migration
+
+```bash
+yarn migration:revert
+```
+
+**New environment variables must be added to Heroku before release.**
+
+### Run unit tests
+
+To run all unit tests
+
+```bash
+yarn test:unit
+```
+
+To have your unit tests running in the background as you change code:
+
+```bash
+yarn test:unit:watch
+```
+
+### Formatting and linting
+
+To run linting
+
+```bash
+yarn lint
+```
+
+To lint and fix
+
+```bash
+yarn lint:fix
+```
+
+Formatting and linting is provided by ESLint and Prettier (see the relevant configs for details).
+
+Workspace settings for VSCode are included for consistent linting and formatting.
+
+## Git flow and deployment
+
+Create new branches from the `develop` base branch. There is no need to run the build command before pushing changes to GitHub, simply push and create a pull request for the new branch. GitHub Actions will run build and linting tasks automatically. Squash and merge feature/bug branches into `develop`.
+
+When changes are ready to be deployed to staging, merge `develop` into `staging`. This will trigger an automatic deployment to the production app by Heroku.
+
+When changes have been tested in staging, merge `staging` into `main`. This will trigger an automatic deployment to the production app by Heroku.
+
+## Swagger
+
+Swagger automatically reflects all of the endpoints in the app, showing their urls and example request and response objects.
+
+To access Swagger simply run the project and visit http://localhost:35001/api/
```

Comment:
`http://localhost:35001/api/` did not work for me.

`http://localhost:35001/swagger/` did.